utils/node_local_test.py

```diff
@@ -1595,6 +1613,189 @@ def run_in_fg(server, conf):
         pass
     dfuse = None
 
+def check_readdir_perf(server, conf):
+    """ Check and report on readdir performance
+
+    Loop over number of files, measuring the time taken to
+    populate a directory, and to read the directory contents,
+    measure both files and directories as contents, and
+    readdir both with and without stat, restarting dfuse
+    between each test to avoid cache effects.
+
+    Continue testing until five minutes have passed, and print
```

Comment:
maybe NLT really is not the place for this.
we should run this on real HW with separate client and server.

Reply:
NLT is the only place we have in CI that does get consistent results across runs, it already runs on real HW and has two nodes available which have been confirmed to have the same performance.

Reply:
No, that's correct.  NLT is local node with tmpfs.  It can run on VMs although it's pointless doing that with this benchmark as you say.  What I meant is there are two separate nodes in CI that can run it, so we have more throughout and redundancy of the testing, but each test only runs on one node.

Reply:
ok  what  i mean by real HW was real storage- pmem & nvme (not VMs vs physical machine). but if the results are consistent as you say then im good with it here.
but this really still supports my argument that NLT should be moved to the testing stage and not kept in the unit test stage. anyway - unrelated to this PR i guess.

Reply:
> ok what i mean by real HW was real storage- pmem & nvme (not VMs vs physical machine). but if the results are consistent as you say then im good with it here.

This would be better but is not available.

> but this really still supports my argument that NLT should be moved to the testing stage and not kept in the unit test stage. anyway - unrelated to this PR i guess.

This part and the fault injection testing would be better in the testing stage, I agree.